<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserListWidget</class>
 <widget class="QWidget" name="UserListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>682</x>
     <y>547</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewOnlineUser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>351</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;
background-color: rgb(255, 255, 255);
border-color: rgb(16, 16, 16);
border: 4px solid rgb(0,0,0);
border-radius : 15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>当前在线用户列表:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheck">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton {
	font: 75 14pt &quot;Segoe Print&quot;;
    background-color: palegoldenrod;
   border-color: rgb(16, 16, 16);
	border: 2px solid rgb(0,0,0);
	border-radius : 15px;
}

QPushButton:hover {
   background-color: khaki;
}

/* Increase the padding, so the text is shifted when the button is
   pressed. */
QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: #d0d67c;
}
</string>
   </property>
   <property name="text">
    <string>查看精灵</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
